### ISSUE TITLE
Dependency problems when importing

### ISSUE THREAD

**Issue by PaBLoX-CL** [typeplate/starter-kit#59]
I've found an issue when trying to import, because while the *chain* is correct in `typeplate-index`, if you try to import only `typeplate-mixins`, results in an error, because it depends on `typeplate-vars`. `typeplate-helpers`, depends on `typeplate-vars` and `typeplate-mixins`.

This of course makes the assumption that everyone would like to import the whole stack, and not parts of it, that while is possible, it's not intuitive at all. So I was wondering if it would be better and more straightforward change the chain dependencies and for example, import `typeplate-vars` inside `typeplate-mixins`.

That could lead to duplication when importing, but since most of those files doesn't generate actual css, there's not going to be duplicate content.

**TL; DR:** the idea is to be able to import each file by itself.

**Comment by grayghostvisuals** [comment]
Thanks for the thoughts and comments. Let me ponder this over and get back to you on this. It's not an unusual or unrealistic request, but will require some thought on how both can be accomplished to satisfy both spectrums of use cases.

**Comment by PaBLoX-CL** [comment]
Great :), if you want I can throw a PR as a proof of concept.

**Comment by grayghostvisuals** [comment]
Instead of a PR can we post some code snippets here? Hate to have you go through the effort and have the PR denied for some weird reason. You can abbreviate if u want with your samples too. Better to discuss it first before the work.
